architecture-building-systems/CityEnergyAnalyst
# 1914 radiation daysim windows installer

This PR fixes #1914 - radiation-daysim now works when installed with the installer. A minimal set of Daysim binaries are shipped and the environment is set up.

To test:
- run the installer https://www.dropbox.com/s/rfka3st83ap07be/Setup_CityEnergyAnalyst_2.13.5a1.exe?dl=0
- using the CEA Console shortcut, run `cea radiation-daysim`
- using the CEA Dashboard shortcut, run Tools -> Energy Potentials -> Solar (Radiation Daysim).

The radiation script should run to completion
